akka-docs/src/main/paradox/scala/stream/stream-quickstart.md

```diff
@@ -241,6 +241,12 @@ whereas `flatMap` would have to operate on streams all the way through.
 
 @@@
 
+@@@ note { title=DIY }
+
+@@fiddle [TwitterStreamQuickstartDocSpec.scala]($code$/scala/docs/stream/TwitterStreamQuickstartDocSpec.scala) { #fiddle_code height=400px extraParams=theme=light&layout=v75 cssStyle=width:100%; }
+
+@@@
```

Comment:
this is also placed in the middle of something else. Move it up above `## Reusable Pieces` and start with

Here is another example that you can edit and run in the browser: